build.sbt

```diff
@@ -50,6 +50,8 @@ lazy val root = project
     coreTestsJVM,
     coreTestsJS,
     docs,
+    examplesJS,
```

Comment:
I think we don't want to include them here (see the note next to their declaration: `Examples sub-project that is not included in the root project.`), otherwise they'll get published.

Reply:
@ghostdogpr, I fixed it, I would like to create an example suite of a real problem (I'm thinking in an Http API embedded into an Environment),  and I didn't create yet an example suite for `TestAspect`, can we postpone the merge?

Reply:
👌